uaclient/entitlements/tests/test_cc.py

```diff
@@ -228,11 +228,11 @@ def exists(path):
         expected_stdout += "\n".join(
             [
                 "Updating package lists",
-                "Installing Canonical Common Criteria EAL2 Provisioning"
+                "Installing Common Criteria EAL2 Provisioning Packages"
                 " packages",
```

Comment:
And this reads extremely weirdly: "Installing Common Criteria EAL2 Provisioning Packages packages"